### DISCUSSION TITLE
The generated route does not have a name

### DISCUSSION THREAD

**Discussion by nigiwen** [posva/unplugin-vue-router#367] [Questions and Help] [answered]
```ts
import { routes } from 'vue-router/auto-routes'

```
![image](https://github.com/posva/unplugin-vue-router/assets/67454874/2b81f5e1-c558-42da-acd1-09bedb8fc8e5)

**Comment by posva** [comment, marked as answer]
The routes that do not have a component won't have a name generated to avoid visiting them by accident. You can always define a name in one of the hooks like extendRoute, extendRoutes, or beforeWrite

**Reply by nigiwen** [reply]
thanks
